src/components/ProjectStatus/index.tsx

```diff
@@ -0,0 +1,244 @@
+import {Image, Linking, Text, View} from 'react-native';
+import React from 'react';
+import {Button} from '../Elements';
+import {ButtonThemes} from '../Elements/Button/types';
+import {usePatchData} from '../../hooks/useQuery';
+import {ENDPOINTS} from '../../config/endpoints';
+import CreateContributionTierForm from '../Forms/ContributionTier/Create';
+import EditProjectForm from '../Forms/Project/Edit';
+import themedStyles from './styles';
+import {useModal} from '../../hooks/useModal';
+import {useTheme} from '../../hooks/useTheme';
+import Contribute from '../Forms/Project/Contribute';
+import PostExclusiveContentsForm from '../Forms/ExclusiveContents/create';
+import successImage from '../../assets/images/success.png';
+import errorImage from '../../assets/images/error.png';
+import {ProjectStatus} from '../Projects/types';
+
+const EditProject = ({projectId}: {projectId: string}) => {
```

Comment:
Please define the component properties types.  It goes for all the components in this file